apps/jupyter/extensions/notebook-runner/src/utils.ts

```diff
@@ -0,0 +1,147 @@
+import { ISessionContext } from "@jupyterlab/apputils";
+import { IKernelConnection } from "@jupyterlab/services/lib/kernel/kernel";
+
+const isPreparedString = "__isPrepared";
+const preparedString = "__prepared";
+const preparedResolveString = "__preparedResolve";
+const preparedRejectString = "__preparedReject";
```

Comment:
should these be called `*String` or `*Key` ? It seems they're used as keys to access some other value, not as strings per-se.